### ISSUE TITLE
[ChartJs] Charta not displayed on first load when on Safari

### ISSUE THREAD

**Issue by cesar-tonnoir** [maestrano/impac-angular#47]
Problem tied to chartjs. May have something to do with https://github.com/nnnick/Chart.js/issues/1622

**Comment by byCedric** [comment]
Does it work if you hover your mouse over the empty canvas? Because I experienced the same with IE10, from the issue you just referenced.

**Comment by cesar-tonnoir** [comment]
@byCedric No: it seems to be working only when I delete and recreate the chart, once the page is fully loaded. It looks very similar to a bug I experienced with ChartJs v1 where it was impossible to draw a chart when the chart div container was invisible (display: none)...

I will investigate this one further with @alexnoox and submit a pull request to ChartJs 2.0 if we find something

**Comment by cesar-tonnoir** [comment]
temporarily fixed by @alexnoox in #50

**Comment by cesar-tonnoir** [comment]
https://maestrano.atlassian.net/browse/IMPAC-119

**Comment by augusto-velez** [comment]
was anyone able to solve the chart problem for safari (not displayed on first load)

**Comment by cesar-tonnoir** [comment]
@augusto-velez: not really: what we do is that we keep looping until the parent element is visible. Then we draw the chart.
Definitely not a perfect solution but is does the job. Check #50 for details.

**Comment by augusto-velez** [comment]
What do you mean that you loop intil the parent is visible, by the parent you mean the iframe? if that how do you check if the iframe is visible.

Thanks for the help
